Foundation/CGFloat.swift

```diff
@@ -949,3 +949,35 @@ extension CGFloat : _CVarArgPassedAsDouble, _CVarArgAligned {
         return native._cVarArgAlignment
     }
 }
+
+extension CGFloat : Codable {
+    @_transparent
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        do {
+            self.native = try container.decode(NativeType.self)
+        } catch DecodingError.typeMismatch(let type, let context) {
+            // We may have encoded as a different type on a different platform. A
+            // strict fixed-format decoder may disallow a conversion, so let's try the
+            // other type.
+            do {
+                if NativeType.self == Float.self {
+                    self.native = NativeType(try container.decode(Float.self))
```

Comment:
@itaiferber Could you double check this for me please?
over at https://github.com/apple/swift/blob/master/stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb#L823 it's
```swift

if NativeType.self == Float.self {
    self.native = NativeType(try container.decode(Double.self))
} else {
    self.native = NativeType(try container.decode(Float.self))
}
```
But shouldn't `Double.self` and `Float.self` be the other way around?